core/src/dird/ua_run.cc

```diff
@@ -525,6 +525,15 @@ int DoRunCmd(UaContext* ua, const char* cmd)
 
     FreeJcr(jcr); /* release jcr */
 
+    /*
+     * For interactive runs we send a message to the audit log
+     */
+    if (jcr->impl->IgnoreLevelPoolOverides) {
```

Comment:
```suggestion
    if (jcr->impl->IgnoreLevelPoolOverrides) {
```

Reply:
Although this typo is not yours, you may want to correct that.

Reply:
corrected spelling error